TravelPoints-BackEnd/src/main/java/com/example/travelpointsbackend/controller/UserWishlistController.java

```diff
@@ -21,6 +22,8 @@ public class UserWishlistController {
     private static final String ADD_FAILED = "{\"message\": \"Add tourist attraction failed!\"}";
     private static final String ADD_SUCCESS = "{\"message\": \"Add tourist attraction success!\"}";
     private static final String GET_FAILED = "{\"message\": \"Get wishlist failed!\"}";
+    private static final String REMOVE_FAILED = "{\"message\": \"Remove tourist attraction failed!\"}";
+    private static final String REMOVE_SUCCESS = "{\"message\": \"Remove tourist attraction success!\"}";
 
```

Comment:
M: as I said to Alina in a previous review, these messages are pointless and in they should be processed in the frontend by the status code